ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__nightly-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -127,7 +127,55 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private
-- as: azure-ipi
+- as: azure-ipi-disconnected-fullyprivate
+  cron: 0 12 */3 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.10 and @azure-ipi and @disconnected'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.10 and @azure-ipi and @disconnected'
+    test:
+    - ref: cucushift-installer-check
+    - chain: openshift-upgrade-qe-test-disconnected
+    workflow: cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate
+- as: azure-ipi-disconnected-fullyprivate-techpreview
```

Comment:
If I remember right, once techpreview feature is enabled, is not recommended to run minor version upgrade, should we only test it in z-stream release? I am thinking if we deserve it even in z-stream upgrade. If we really want to cover it, maybe give it low frequency.

Reply:
I was not aware of it. Thanks for letting me know. I'm fine to drop it. 
Btw, is it possible to add some content if upgrade is not supported in the documentation in the workflow or in our CI unifying spreadsheet? Either is fine.

Reply:
Let us drop it for now. I just noticed it this morning in https://coreos.slack.com/archives/CPJQCSN49/p1657231537322049.

I also added a new column named "upgrade supported" in our CI unifying spreadsheet

Reply:
Thanks! Dropped the tests for techpreview.